test/specs/inspecting.test.js

```diff
@@ -139,7 +139,7 @@ module.exports = (fetchMock) => {
 						name: 'my-route'
 					});
 
-				await fm.fetchHandler('http://it.at.here/');
+				await fm.fetchHandler('http://it.at.here');
```

Comment:
I feel it'd be better to keep ending slashes consistent within a single test, and only vary them in tests that explicitly test the trailing slash behaviour